src/v/kafka/protocol/schemata/describe_configs_request.json

```diff
@@ -16,11 +16,13 @@
 {
   "apiKey": 32,
   "type": "request",
+  "listeners": ["zkBroker", "broker"],
```

Comment:
> k/p/schemata: update describe configs req & resp

in the commit message it's good to add a note about which version of kakfa you used to grab the updated json files.